File: Manual 03/Question05.cpp
#include <iostream>
using namespace std;

int main()
{
    // Checked Board
    int i, j, k, n, c;
    cout << "Enter number of rows: ";
    cin >> n;
    cout << "Enter number of columns: ";
    cin >> c;

    i = 1;
    while (i <= n)
    {
        k = 1;
        while (k < 2)
        {
            if (i % 2 != 0)
            {
                cout << " ";
            }
            k++;
        }

        j = 1;
        while (j <= c)
        {
            cout << "-";
            j++;
        }
        cout << endl;
        i++;
    }
}

File: Python/Manual 03/Question05.py
def print_checked_board(n, c):
    i = 1
    while i <= n:
        if i % 2 != 0:
            print(" ", end="")

        j = 1
        while j <= c:
            print("-", end="")
            j += 1
        print()
        i += 1


n = int(input("Enter number of rows: "))
c = int(input("Enter number of columns: "))
print_checked_board(n, c)
